scripts/specs.ts

```diff
@@ -58,6 +50,14 @@ testIsOK();
 let checked = 0;
 let errors = 0;
 
+for (const [allowedUrl, message] of defaultAllowlist) {
+    if (specUrls.some(specUrl => specUrl.href === allowedUrl)) {
```

Comment:
I think it's best to use `isOK` here to provide stronger guarantees that none of the URLs in `defaultAllowlist` are OK because they match something in `specUrls`. I think this would do it:

```suggestion
    if (isOK(allowedUrl, [])) {
```

And a comment before the for-loop like "Ensure every exception in defaultAllowlist is needed."